Worktez/worktez
# Fix(my-organization): fixed my organization functinalities

Signed-off-by: Abhishek Mishra [am1426620@gmail.com](mailto:am1426620@gmail.com)
### Functionality:
fixing the UI of my organization page and its upload logo feature

### Solution:
fixed UI of my org in dark theme and fixed the functionalities for updating organization logo.
also did some necessary changes in start service and firebase indexing

### Risk level:

### How to test:
try the my organization page in dark mode and also try to upload a logo and check in firestore
